test/routes/author/index.spec.js

```diff
@@ -0,0 +1,71 @@
+import assert from 'power-assert';
+import request from 'supertest';
+import app from 'app.js';
+import AutherModel from 'models/Author';
+
+describe('/routes/author', ()=> {
+  let response;
+
+  before((done)=> {
+    const author = new AutherModel({
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).